tests/sentry/monitors/test_tasks.py

```diff
@@ -109,6 +113,49 @@ def test_missing_checkin(self, mark_environment_missing_mock):
         ).replace(second=0, microsecond=0)
         assert missed_checkin.monitor_config == monitor.config
 
+    @mock.patch("sentry.monitors.tasks.mark_environment_missing")
+    def test_temp_ingore_next_checkin_equal_latest(self, mark_environment_missing_mock):
+        org = self.create_organization()
+        project = self.create_project(organization=org)
+
+        task_run_ts, ts = make_ref_time()
+
+        monitor = Monitor.objects.create(
+            organization_id=org.id,
+            project_id=project.id,
+            type=MonitorType.CRON_JOB,
+            config={
+                "schedule": "* * * * *",
+                "schedule_type": ScheduleType.CRONTAB,
+                "max_runtime": None,
+                "checkin_margin": None,
+            },
+        )
+
+        # This monitor will be ignored, the next_checkin and
+        # next_checkin_latest are equal. In the future this should never happen
+        # since the `checkin_margin` will have a minimum of `1`.
+        monitor_environment = MonitorEnvironment.objects.create(
+            monitor=monitor,
+            environment=self.environment,
+            last_checkin=ts - timedelta(minutes=1),
+            next_checkin=ts,
+            next_checkin_latest=ts,
```

Comment:
just to confirm, these get pulled out of the db with `UTC` timezone info, right?

Reply:
I guess it doesn't matter because we're using a `filter` now